dpe/src/commands/mod.rs

```diff
@@ -121,6 +120,27 @@ pub struct CommandHdr {
 
 impl CommandHdr {
     const DPE_COMMAND_MAGIC: u32 = u32::from_be_bytes(*b"DPEC");
+
+    pub fn new_for_test(command: Command) -> CommandHdr {
+        let cmd_id = match command {
+            Command::GetProfile => Command::GET_PROFILE,
+            Command::InitCtx(_) => Command::INITIALIZE_CONTEXT,
+            Command::DeriveChild(_) => Command::DERIVE_CHILD,
+            Command::CertifyKey(_) => Command::CERTIFY_KEY,
+            Command::Sign(_) => Command::SIGN,
+            Command::RotateCtx(_) => Command::ROTATE_CONTEXT_HANDLE,
+            Command::DestroyCtx(_) => Command::DESTROY_CONTEXT,
+            Command::ExtendTci(_) => Command::EXTEND_TCI,
+            Command::TagTci(_) => Command::TAG_TCI,
+            Command::GetTaggedTci(_) => Command::GET_TAGGED_TCI,
+            Command::GetCertificateChain(_) => Command::GET_CERTIFICATE_CHAIN,
+        };
+        CommandHdr {
+            magic: Self::DPE_COMMAND_MAGIC,
+            cmd_id,
```

Comment:
nit: If you implement this as the From trait on Command, you can probably just use `command.into()`

```rust
impl From<u32> for Command {
...
}
```

For example https://github.com/chipsalliance/caliptra-sw/blob/7a3bb74f9c3c8131f4f20bbe65bb7bfffb89a490/cpu/src/trap.rs#L100